sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventProcessor.java

```diff
@@ -4,31 +4,22 @@
 package com.azure.messaging.eventhubs;
 
 import com.azure.core.util.logging.ClientLogger;
-import com.azure.messaging.eventhubs.models.PartitionContext;
-import com.azure.messaging.eventhubs.models.PartitionOwnership;
-import com.azure.messaging.eventhubs.models.EventHubConsumerOptions;
+import com.azure.messaging.eventhubs.implementation.PartitionBasedLoadBalancer;
+import com.azure.messaging.eventhubs.implementation.PartitionPumpManager;
 import com.azure.messaging.eventhubs.models.EventPosition;
-
-import java.io.IOException;
-import java.util.Map;
-import java.util.Objects;
 import java.util.UUID;
-import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicBoolean;
-
-import org.reactivestreams.Publisher;
 import reactor.core.Disposable;
-import reactor.core.publisher.Flux;
 import reactor.core.scheduler.Scheduler;
 import reactor.core.scheduler.Schedulers;
 
 /**
  * This is the starting point for event processor.
  * <p>
- * Event Processor based application consists of one or more instances of {@link EventProcessor} which are
- * set up to consume events from the same Event Hub + consumer group and to balance the workload across different
- * instances and track progress when events are processed.
+ * Event Processor based application consists of one or more instances of {@link EventProcessor} which are set up to
```

Comment:
nit:  This reads a bit awkward to me.  Maybe consider something like "... which have been configured to consume events..."